controller.go

```diff
@@ -686,13 +724,18 @@ func (c *Controller) handleObject(obj interface{}) {
 	}
 }
 
-func makeUserdata(authToken string) string {
+func makeUserdata(authToken string, useSudo bool) string {
```

Comment:
`makeUserdata` changed since your PR, please can you check and rebase? It now supports the "pro" version of inlets.